shard/shard.go

```diff
@@ -20,10 +20,24 @@
 
 package shard
 
+const (
+	// INITIALIZING represents a shard newly assigned to an instance
+	INITIALIZING State = iota
+
+	// AVAILABLE represends a shard bootstraped and ready to serve
+	AVAILABLE
```

Comment:
Let's stick with our current const naming scheme, this should just be `Available`.